src/base/utils/fs.cpp

```diff
@@ -49,6 +49,13 @@
 #include <sys/vfs.h>
 #endif
 
+#include <cstring>
+#include <sys/stat.h>
+#include <sys/types.h>
```

Comment:
the above 2 are unix-only headers right? should  you move them in `#ifdef Q_OS_UNIX`?

Reply:
>the above 2 are unix-only headers right? 

No. Otherwise the code couldn't compile on Windows.

Reply:
>No. Otherwise the code couldn't compile on Windows.

I just saw MSVC have this header... then it should move up, above `#if defined(Q_OS_WIN)` section.